src/reaper_osara.cpp

```diff
@@ -2285,17 +2285,19 @@ bool maybeFixTabInSaveDialog(bool previous) {
 // Handle keyboard keys which can't be bound to actions.
 // REAPER's "accelerator" hook isn't enough because it doesn't get called in some windows.
 LRESULT CALLBACK keyboardHookProc(int code, WPARAM wParam, LPARAM lParam) {
-	if (code != HC_ACTION && wParam != VK_APPS && wParam != VK_RETURN &&
-			wParam != VK_F6 && wParam != 'B' &&
-			wParam != VK_TAB) {
+	const bool isKeyDown = !(lParam & 0x80000000);\
+	if (!isKeyDown || code != HC_ACTION || (
```

Comment:
1. On HC_ACTION: Oops. My bad. :) Nice catch.
2. Are you absolutely certain there are no cases left in this code that want a key up? I know there was at least one obscure key up case at one point, but maybe that got eliminated with some refactor (perhaps the context menu refactor).